content/blog/announcing-crossguard-preview/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Announcing Crossguard Preview"
+date: 2019-12-02
+draft: true
+meta_desc: "Today we are announcing Pulumi CrossGuard, a Policy as Code solution, is available to all Team Pro and Enterprise customers to preview."
+meta_image: crossguard.svg
+authors: ["erin-krengel"]
+tags: ["crossguard", "policy-as-code", "New-Features", "Pulumi-News", "Features"]
+---
+
+Over the past few months, we have been hard at work on Pulumi CrossGuard, a Policy as Code solution. Using the `@pulumi/policy` library, you can express flexible business and security rules using code. CrossGuard enables organization administrators to enforce these policies across their organization or just on specific stacks. CrossGuard allows you to verify or enforce custom policies on changes before they are applied to your resources.
```

Comment:
Mainly want to avoid making this look like a JavaScript-only feature:

```suggestion
Over the past few months, we have been hard at work on Pulumi CrossGuard, a Policy as Code solution. Using CrossGuard, you can express flexible business and security rules using code. CrossGuard enables organization administrators to enforce these policies across their organization or just on specific stacks. CrossGuard allows you to verify or enforce custom policies on changes before they are applied to your resources.
```

Reply:
It might be nice to add a sentence like this also:

> CrossGuard is 100% open source and available to all users of Pulumi, including the Community Edition. Advanced organization-wide policy management features are available to Team Pro and Enterprise customers.